test/v4/retrieveAnnualSubmission/def2/model/response/Def2_RetrieveAnnualSubmissionResponseSpec.scala

```diff
@@ -29,8 +28,6 @@ class Def2_RetrieveAnnualSubmissionResponseSpec
     with Def2_RetrieveAnnualSubmissionFixture
     with MockSeBusinessFeatureSwitches {
```

Comment:
Please remove `with MockSharedAppConfig with MockSeBusinessFeatureSwitches` as it is not required, thanks.